### PR TITLE
Remove all instances of location.query and fix some other bugs

### PR DESCRIPTION
**Pull request by erlingfn**
Change some of the old react router v3 code that slipped through the cracks. Should fix this [issue](https://github.com/webkom/lego/issues/1853)

### CODE REVIEW

File: app/routes/surveys/index.js [hunk 1]
```diff
@@ -44,7 +44,6 @@ const surveysRoute = ({ match }: { match: { path: string } }) => (
           Component={SurveyDetailRoute}
         />
         <RouteWrapper
-          exact
```
Comment (LudvigHz) [inline app/routes/surveys/index.js:47]: Why did you remove `exact` here? If the idea is that the above route matches first, I still like the explicit handling to avoid unexpected behaviour.

Comment (erlingfn) [reply app/routes/surveys/index.js:47]: Thought it had something to do with urls with query params not being matched, but it turned out to be the templates not being in the backend fixtures. Will add it back in
